contributing/getting-started-quick.md

```diff
@@ -78,6 +78,7 @@ This Quick Start guide is the TL;DR version of the longer [end-to-end guide](get
      outdated).
      
      ```shell
+     git fetch upstream master
```

Comment:
Isn't this covered by point 11 above?

Reply:
Hi Cathy.
Not quite. The step outlines what needs to be done but it doesn't specify how to go about it i.e fetching the ref.
For reference purposes, please have a look at [set up aliases for the remote Git repositories](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#set-up-aliases-for-the-remote-git-repositories)
Happy to fetch my fork and demo this.

Reply:
> 11.  Fetch data from the `upstream` remote:
      ```          
        git fetch upstream
      ```	     

Let's just change `git fetch upstream` to `git fetch upstream master`. The latter is better because a plain fetch would fetch a bunch of unnecessary tags from every release we ever did. There might have been a bit of a race, since an earlier PR added point 11 precisely for this reason.

Reply:
Hi @cathyxz I'm sorry I hadn't taken note of step 11. As you rightly put it, this change is appropriate for step 11 and not 12. I'll change this. My apologies for the oversight.

Reply:
No worries. I only remembered this because I reviewed https://github.com/ampproject/amphtml/pull/18289 for this change.